.gitignore

```
@@ -23,4 +23,4 @@ packages/*/out
 *.vsix
 
 # Test Artifacts
-test/workspace/.vscode
+# test/workspace/.vscode
```

I think this can just be removed entirely :) 
Otherwise, could do:
```suggestion
test/workspace/.vscode
!test/workspace/.vscode/settings.json
```